src/verilator/README.md

```diff
@@ -0,0 +1,26 @@
+Basic Verilator Wrapper
+* ITop is intended as a wrapper for the top level module of your design
+* IMem is intended as a wrapper for a blackbox verilog memory module
+	-You should use IMem to forward all memory requests to gem5 rather than the c++ model
+* Verilator Object is a basic implementation of ITop for the riscv sodor 1 stage core
+* IMem is currently a work in progrress
+
+Building
+* First you must generate your verilator c++ models, and include them as needed
+	-add flags to the SConscript to modify the compiler flags and include wha tyou need (.i.e. env.Append(CCFLAGS='-I ./<directory>'))
```

Comment:
I don't understand why this is needed. I think there's a better scons solution here... But let's not worry about that this week.